pl-faded-parsons.py

```diff
@@ -222,6 +222,7 @@ def get_code_lines():
     html_params.update({
         "given" : given,
         "uuid": pl.get_uuid(),
+        "previous_log" : data['submitted_answers'].get('log', "[]")
```

Comment:
This adds to the existing bug referenced in #39 - I believe the real dictionary key would be `answers_name + 'log'` when answers_name is set (it should be required but we don't do that yet).